release_notes/44-release-notes.adoc

```diff
@@ -561,6 +562,27 @@ The issue occurred when more than 10 collections were created and a vulnerabilit
 +
 This update ensures that the linked collection is retained, avoiding the error message.
 
+
+[id="resolved-in-version-441_{context}"]
+=== Resolved in version 4.4.1
+
+*Release date*: 15 April 2024
+
+* With this update, rendering the Helm charts `stackrox-central-services` and `stackrox-secured-cluster-services` requires a Helm version 3.9.0 or later, as support for older Helm versions has been discontinued.
+
+* Fixed an issue where {product-title-short} would not qualify the registry and path of images from the container runtime if the `ROX_UNQUALIFIED_SEARCH_REGISTRIES` environment variable was set to `true` on both Central and Sensor. This update now enables support for CRI-O's unqualified search registries and short name aliases.
```

Comment:
I thought we do not document this - @dcaravel to verify whether this should be here or not.

Reply:
That's correct, please remove the `ROX_UNQUALIFIED_SEARCH_REGISTRIES` section for now. Thanks @agantony

Reply:
ack